test/queries/queries.js

```diff
@@ -0,0 +1,321 @@
+const temporal = {
+    "query": {
+        "bool": {
+            "must": {
+                "match_all": {}
+            },
+            "filter": [
+                {
+                    "range": {
+                        "metadata.o2r.temporal.begin": {
+                            "from": "2015-01-01T00:00:00.000Z"
+                        }
+                    }
+                },
+                {
+                    "range": {
+                        "metadata.o2r.temporal.end": {
+                            "to": "2016-01-02T00:00:00.000Z"
+                        }
+                    }
+                }
+            ]
+        }
+    },
+    "from": 0,
+    "size": 10
+};
+
+const europe = {
+    "query": {
+        "bool": {
+            "must": {
+                "match_all": {}
+            },
+            "filter": {
+                "geo_shape": {
+                    "metadata.o2r.spatial.union.geojson.geometry": {
+                        "shape": {
+                            "type": "polygon",
+                            "coordinates":
+                                [
+                                    [
+                                        [
+                                            -7.294921874999999,
+                                            38.54816542304656
+                                        ],
+                                        [
+                                            34.013671875,
+                                            38.54816542304656
+                                        ],
+                                        [
+                                            34.013671875,
+                                            68.6245436634471
+                                        ],
+                                        [
+                                            -7.294921874999999,
+                                            68.6245436634471
+                                        ],
+                                        [
+                                            -7.294921874999999,
+                                            38.54816542304656
+                                        ]
+                                    ]
+                                ]
+                        },
+                        "relation": "within"
```

Comment:
all test queries use "within", please add at least one query using another relation, so that we can also refer the client implementers to the tests as examples.